analysis/simulations/pan_tedsim/README.rst

```diff
@@ -12,5 +12,4 @@ File description
     `run.py` | Main script to run the moslin/`LineageOT <https://www.nature.com/articles/s41467-021-25133-1>`_ analysis.
     `utils_analysis.py` | Utilities for data visualization in `ZP_2023-04-02_tedsim_analysis.ipynb`.
     `utils_run.py` | Utilities for `run.py`.
-    `ZP_2023-04-02_tedsim_analysis.ipynb` | Visualize the results - TedSim data and performance. Uses a pre-computed TedSim simulation to visualize the initial state tree, simulated tree and gene expression.
-    Imports the grid search results and visualizes the cost as a function of method and stochastic silencing rate. 
+    `ZP_2023-04-02_tedsim_analysis.ipynb` | Visualize the results - TedSim data and performance. Uses a pre-computed TedSim simulation to visualize the initial state tree, simulated tree and gene expression. Imports the grid search results and visualizes the cost as a function of method and stochastic silencing rate. 
```

Comment:
I think you need "``"

Reply:
i prefer files not marked like @michalk8 had it for other readmes.